gadk/elements.py

```diff
@@ -111,6 +141,14 @@ def __init__(
         self._action = action
         self._with: Dict[str, str] = with_args or {}
 
+    def __repr__(self):
+        slug = (
+            f"id={self._id!r}" if self._id is not None
```

Comment:
Why use `!r` rather than `=` here as above?

Reply:
Good spot - I'll change that.